### ISSUE TITLE
add appropriate pauses

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#260]
I don't want to do this "soon", because there are a bunch of upcoming UI changes that will alter what the appropriate pauses should be. Also, I can only do one thing at a time (until I start hiring / paying contractors). OTOH, I've been aware of this since day one so I may as well make an issue out of it so other people can see that I'm aware.

I do, however, plan on fixing it so people can scroll-backward (#257)  without fighting with my poorly implemented autoscroll.

**Comment by ctm** [comment]
Mr. Meck (on June 23rd):

> [...] I think it would be very nice, from an entertainment standpoint, to have pauses when dealing out the final cards of a hand in an all-in situation.  Currently, once we reach that situation and no more betting is possible, mb2 instantaneously deals out the rest of the hand and you have to scroll up during the next hand to see what happened (and in stud games, it's very hard to see what the final few cards were).  [...].

> Especially for the actual BARGE but in general, pauses to deal out the remaining cards would be great.  Players have time to figure out what their outs are and to trash talk each other, pray to the poker gods (I'm a poker atheist), etc., either in the chat box or on Zoom if the players are conferencing.  It'll be much more satisfying to watch [BARGEr] crush [other BARGER] slowly rather than in a blink of an eye!

**Comment by ctm** [comment]
I've added a.5 second pause at the end of every hand where nobody busts and a 15 second pause after at least one player has bust.

I've also added a 5 second pause once there's no more action due to all-ins (w/no draws coming) and an additional 5 second pause at the end of each round when there's no action to come. So that means there's a 10 second pause when everyone goes all-in, oh and the all-in's cards are revealed.

I'm going to deploy this for this evening's game, but make it so I can roll back to a previous version if it causes any trouble.
